primitives/runtime/src/generic/unchecked_extrinsic.rs

```diff
@@ -161,6 +161,22 @@ where
 			None => CheckedExtrinsic { signed: None, function: self.function },
 		})
 	}
+
+	#[cfg(feature = "try-runtime")]
+	fn unchecked_into_checked(
```

Comment:
not super happy about having the `try-runtime` leaked into this crate.

Can we make it `unsafe fn into_unchecked()` instead and remove the feature?
Then document that the signature of the transaction is not verified.

Reply:
I don't understand why this is needed FWIW.

Reply:
https://github.com/paritytech/substrate/pull/12537/files#diff-71647e596916530d84fd02546feb833ff703dbf19397d076c499ab7dbf265da2R252 is where this is needed, to disable signature verification.